mangaki/mangaki/utils/common.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from mangaki.utils.chrono import Chrono
+import pickle
+import os.path
+
+
+PICKLE_DIR = os.path.join(settings.BASE_DIR, '../pickles')
+
+
+class RecommendationAlgorithm:
+    nb_users = None
```

Comment:
AFAIK, this kind of declaration are for **class* attributes rather than instance attributes.
Is that intended ?